cases/index.html

```diff
@@ -1,128 +1,297 @@
+                </li>
+                <li>
+                    We have integrated with various of Fischer Lighting's other suppliers and tools, and built custom support for supply handling and planning of complex and interdependent products; advanced requirements specification of Fischer Lighting's customers' needs; project planning and execution; and report generation with advanced calculations. We have migrated data from a complex legacy system, and launched a new system that significantly saves employee work time, freeing up time for more important tasks.
+                </li>
+            </ul>
+            <button disabled class="case__read-more">More elaborate case coming soon&hellip;</button>
+        </div>
+    </div>
+
+    <div class="case">
+        <div class="case__meta">
+            <div class="case__meta-intro">
+                <h1>
+                    <a href="https://kbharkiv.dk/" target="_blank" rel="noopener">Københavns Stadsarkiv</a>
+                </h1>
+                <img src="/img/kbh-stadsarkiv.png" class="case__logo">
+            </div>
+            <div class="case__icon">🖼️</div>
+            <div class="case__icon">🔎</div>
+            <div class="case__icon">⚙️</div>
+        </div>
+        <div class="case__content">
+            <h1>Summary</h1>
+            <ul>
+                <li>
+                    Københavns Stadsarkiv needed a new contractor to take over maintenance for <a href="https://kbhbilleder.dk/" target="_blank" rel="noopener">kbhbilleder.dk</a>, a public-facing system covered by public sector regulations. Some of these regulations also meant that the system needed an accessibility audit and changes made to ensure it lived up to the required standards.
+                </li>
+                <li>
+                    In addition to the accessibility audit and update (and various small changes or features, we have planned and executed in cooperation with them), we also performed a full review of the system, and helped plan updates to the maintainability, stability and security, all while balancing impact with budget.
+                </li>
+                <li>
+                    Concurrently, we have been working with Københavns Stadsarkiv on <a href="https://link-lives.dk/" target="_blank" rel="noopener">Link-Lives</a>, a research project with many stakeholders aiming to make available digital versions of historical records about Danes. On this project, we have been working closely with other employees at Løbenhavns Stadsarkiv, including their own software developers and designers. We have taken ownership of the public-facing interface of the system, but coordinate closely and provide our expertise in advanced systems engineering and infrastructure automation, as well. The project is scheduled to launch in 2021.

```

Comment:
The project is scheduled to launch in 2021. => The project is scheduled to launch in the third quarter of 2021.

Reply:
Jeg ser ingen grund til at være ekstra præcis?